corehq/apps/couch_sql_migration/tests/test_migration.py

```diff
@@ -633,7 +633,6 @@ def test_migrate_attachments(self):
             </n1:meta>
         </data>""".format(
             date='2016-03-01T12:04:16Z',
-            attachment_source=attachment_source,
```

Comment:
I guess this is just cleanup since it looks like `{attachment_source}` was never referenced in the string template?